docs/development/Building in Arch Linux.md

```diff
@@ -0,0 +1,56 @@
+# Building in Arch Linux
+
+To build BetaFlight, first the ARM toolchain has to be installed:
+
+## Setup GNU ARM Toolchain
+
+```bash
+pacman -S arm-none-eabi-gcc arm-none-eabi-newlib arm-none-eabi-binutils
+```
+
+For running tests, the `libblocksruntime` package from the AUR also has to be installed:
+``` bash
+trizen -S libblocksruntime
+```
+
+At the time of writing, the repository version of GCC is 9.2.0, unlike 9.2.1 which BetaFligh expects.
+If you get the following error, you can export an override in your shell before building:
```

Comment:
While the shell override will work it is cumbersome. It is easier to follow the instructions in the error message and add the workaround to `make/local.mk`.